actors/migration/nv9/util.go

```diff
@@ -122,3 +123,22 @@ func migrateHAMTAMTRaw(ctx context.Context, store cbor.IpldStore, root cid.Cid,
 	}
 	return store.Put(ctx, outRootNodeOuter)
 }
+
+// Guards an unsychronized store with a mutex.
```

Comment:
I think this is redundant with support/ipld/store.go `NewSyncADTStore`

Reply:
Ah, I needed this for my ad-hoc tests which I hacked into the scenario test VM (which does not use a sync store), but yes we don't need it for production.